spec/controllers/pxe_controller_spec.rb

```diff
@@ -73,4 +73,22 @@
       expect(assigns(:edit)[:new][:log_password]).to eq(ps.authentication_password(:default))
     end
   end
+
+  describe "#tree_select" do
+    before do
+      login_as FactoryGirl.create(:user_admin)
+    end
+
+    subject do
+      post :tree_select, :params => {:id => 'root'}
+    end
+
+    context "render pxe partial" do
```

Comment:
This context doesn't seem required, you can omit it and instead use a more descriptive string in the `it`.